src/operator/src/statement.rs

```diff
@@ -338,6 +345,31 @@ impl StatementExecutor {
             "DATESTYLE" => set_datestyle(set_var.value, query_ctx)?,
 
             "CLIENT_ENCODING" => validate_client_encoding(set_var)?,
+            // TODO: write sqlness test for query timeout variables
```

Comment:
An integration test that is mysql only is better. Let sqlness focus more on sql correctness.

Reply:
just learned `-- SQLNESS PROTOCOL MYSQL` can trigger mysql channel very useful.
In commit 3: removed the old comment and added basic set and show mysql only test, wonder if we have additional tools to trigger a actual timeout in test (I was think about `SELECT SLEEP(3);`  but its not supported yet).